Manager/Doctrine/AccessTokenManager.php

```diff
@@ -26,7 +26,10 @@ public function __construct(EntityManagerInterface $entityManager)
      */
     public function find(string $identifier): ?AccessToken
     {
-        return $this->entityManager->find(AccessToken::class, $identifier);
```

Comment:
This change seems completely unrelated with Symfony 5.

Reply:
i'm happy to revert those changes but ... are you sure that you want me to? the arguments were referencing those classes as "string paths" and changing those references to the ":class" syntax promotes better static analysis, which is immensely helpful when undertaking framework upgrades like this one. It definitely helped me in the process of upgrading for compatibility for symfony 4.4/5.0.

Reply:
But this change (adding a temporary variable just to add a docblock) doesn't really improve anything (other than maybe making your IDE happy as technically the `find` method returns `object|null`) . The method return typehint is there, the entityManager gets called with `AccessToken::class` and there should also be tests for this as we have ~90% test coverage (if there isn't there should be one). So I don't see how this change helped you with the Symfony 5 upgrade, especially since this change is just a docblock change which concerns a Doctrine, not a Symfony class.

Reply:
oh okay yes sure, I wasn't looking at the correct change. ok thank you.

Reply:
ok should be fixed now ...